sycl/doc/extensions/proposed/sycl_ext_oneapi_graph.asciidoc

```diff
@@ -895,10 +895,16 @@ Table 10. Queue info query
 
 |===
 
-A default constructed event is returned when a user submits a command-group to
-a queue in the recording state. These events have status
-`info::event_command_status::complete` and a user waiting on them will return
-immediately.
+Events returned from queue submissions when a queue is in the recording state
+may only be used as parameters to `handler::depends_on()` or as dependent
+events for queue shortcuts like `queue::parallel_for()` for submissions which
+are being recorded to the same modifiable `command_graph`.
+
+Waiting on an event returned from a queue submission recorded to a graph
+will throw synchronously with error code `invalid`.
+
+Calling `queue::wait()` on a queue in the recording state is an error and
```

Comment:
I think we're going to need to update the "7.9. Error Handling" section too, it has wording

> While a queue is in the recording state, methods performed on that queue which are not command submissions behave as normal. This includes waits, throws, and queries on the queue. These are all ignored by the graph system, as opposed to throwing an exception when in queue recording mode. This is because otherwise there would be no thread safe way for a user to check they could call these functions without throwing, as a query about the state of the queue may be immediately stale.

Reply:
Ah apologies I missed that section! I've updated it now. Let me know if it makes sense to you!